.github/workflows/quality.yml

```diff
@@ -23,12 +24,56 @@ jobs:
         run: |
           ./gradlew ktlintCheck --continue
       - name: Upload reports
+        if: always()
         uses: actions/upload-artifact@v2
         with:
           name: ktlinting-report
-          path: vector/build/reports/ktlint/*.*
+          path: |
+            */build/reports/ktlint/ktlint*/ktlint*.txt
+      - name: Handle Results
+        if: always()
+        id: get-comment-body
+        run: |
+          results="$(cat */*/build/reports/ktlint/ktlint*/ktlint*.txt */build/reports/ktlint/ktlint*/ktlint*.txt | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g")"
+          if [ -z  "$results" ]; then
+            body="👍 ✅ 👍"
+          else
+            body="\`👎 ❌ 👎 ${results}\`"
+          fi
+          body="${body//'%'/'%25'}"
+          body="${body//$'\n'/'%0A'}"
+          body="${body//$'\r'/'%0D'}"
+          body="$( echo $body | sed 's/\/home\/runner\/work\/element-android\/element-android\//\`<br\/>\`/g')"
+          body="$( echo $body | sed 's/\/src\/main\/java\// 🔸 /g')"
+          body="$( echo $body | sed 's/im\/vector\/app\///g')"
+          body="$( echo $body | sed 's/im\/vector\/lib\/attachmentviewer\///g')"
+          body="$( echo $body | sed 's/im\/vector\/lib\/multipicker\///g')"
+          body="$( echo $body | sed 's/im\/vector\/lib\///g')"
+          body="$( echo $body | sed 's/org\/matrix\/android\/sdk\///g')"
+          body="$( echo $body | sed 's/\/src\/androidTest\/java\// 🔸 /g')"
+          echo "::set-output name=body::$body"
+      - name: Find Comment
+        if: always()
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Ktlint Results
+      - name: Publish ktlint results to PR
+        if: always()
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ### Ktlint Results
 
-# Lint for main module and all the other modules
+            ${{ steps.get-comment-body.outputs.body }}
+          reactions: rocket
```

Comment:
I do not see the added value to automatically add a reaction to the generated comment (except that it is nice to know that it is possible to do so)

Reply:
haha yes I just wanted to test it, I will remove it